studio/src/rte/rte-field.js

```diff
@@ -515,6 +515,7 @@ class RteField extends LitElement {
             title,
             target: target || '_self',
             class: variant || 'primary-outline',
+            tabIndex: '0',
```

Comment:
Are we okay with leaving this change @yesil? Or you will handle it in Milo web component code?